Added Image Registry and ChaosHub docs (#79)
litmuschaos/litmus-docs: website/docs/concepts/chaoshub.md

sidebar_label: ChaosHub
---

---

> ## Introduction

Reviewer: Intro may not have a heading called intro...you can just start with the intro paragraph

> ChaosHub allows a user to orchestrate workflows from the Public **[ChaosHub](http://hub.litmuschaos.io/)** or an alternate source for the Experiments (basically, a **[fork](https://github.com/litmuschaos/chaos-charts)** of the public hub with custom experiments).

Reviewer 2: We should use the second person to address, as this convention is used everywhere. You can write - `ChaosHub allows a user` -> `ChaosHub allows a you`

## Prerequisites

> The following should be required before creating a Chaos Workflow:

Reviewer 2: following should -> following are

> The following should be required before creating a Chaos Workflow:

Reviewer 3: ```suggestion
The following are the prerequisites for creating a Chaos Workflow:
```

`should` indicates a possibility.. but it is in fact a mandate in this case..

- [ChaosCenter](chaos-center)
- [ChaosAgent](agent)
- Fork of [Chaos-Charts](https://github.com/litmuschaos/chaos-charts) repository  

:::note
  An active internet connection is required to clone the git repository for the first time installation.
:::

## Connecting a Git repository using ChaosHub

> With ChaosHub, a user can construct chaos workflows by selecting, tuning and sequencing different experiments together from their connected ChaosHubs.

Reviewer 2: user -> you

> User can make changes in their forked repositories and sync it with the Portal to get the latest changes from the fork.

Reviewer 2: Same here
